repo2docker/buildpacks/python/__init__.py

```diff
@@ -117,27 +117,36 @@ def get_assemble_scripts(self):
         # will be installed in python3 venv. This is less of a
         # surprise than requiring python2 to be requirements2.txt tho.
         assemble_scripts = super().get_assemble_scripts()
+        setup_py = self.binder_path('setup.py')
```

Comment:
This will find a `setup.py` which is in a `binder/` sub-directory. I think we should not pick that up. Not sure when such a file would exist but with enough repos eventually we will find one that does have a `binder/setup.py`.